db/seeds.rb

```diff
@@ -1,8 +1,9 @@
-admin_role = Role.find_or_create_by(name: "admin")
 
 grove = Grove.create!(name: "Aspen")
 Grove.create!(name: "Fir")
 Grove.create!(name: "Pine")
+Role.create!(name: "teacher")
```

Comment:
I would say a user is a teacher if they are a Teacher, and for the roles drop down provide a blank, so you don't have to add the role. Does that make sense?